### PR TITLE
Char loadout balaclava

### PR DESCRIPTION
**Pull request by CapCamIII**
# About the pull request

Adds the balaclava and green balaclava as loadout items, also updates green balaclava's sprite.

# Explain why it's good for the game

Interesting loadout item, works for all factions, looks unique(its not a coif, or a reskin of coif like USCM Balaclava)

image

Also one step closer to this absolute beast of a look ^

# Testing Photographs and Procedure

<details>

image

image

image

image

</details>

# Changelog

:cl:
add: Adds 2 new char setup loadout items, a black and green balaclava, 2 points each.
fix: Balaclava object sprite has a correct offset now.
/:cl:

### CODE REVIEW

File: code/modules/clothing/masks/miscellaneous.dm [hunk 1]
```diff
@@ -66,20 +66,21 @@
 
 
 /obj/item/clothing/mask/balaclava
-	name = "balaclava"
-	desc = "LOADSAMONEY"
+	name = "Balaclava"
```
Comment (stanalbatross) [inline code/modules/clothing/masks/miscellaneous.dm:69]:
```suggestion
	name = "balaclava"
```

File: code/modules/clothing/masks/miscellaneous.dm [hunk 2]
```diff
@@ -66,20 +66,21 @@
...
+	flags_inventory = COVERMOUTH|ALLOWREBREATH|ALLOWCPR
+	flags_inv_hide = HIDEFACE|HIDEALLHAIR|HIDEEARS
+	flags_cold_protection = BODY_FLAG_HEAD
+	min_cold_protection_temperature = ICE_PLANET_min_cold_protection_temperature
 	w_class = SIZE_SMALL
 
 /obj/item/clothing/mask/balaclava/tactical
-	name = "green balaclava"
-	desc = "Designed to both hide identities and keep your face comfy and warm."
+	name = "Green Balaclava"
```
Comment (stanalbatross) [inline code/modules/clothing/masks/miscellaneous.dm:80]:
```suggestion
	name = "green balaclava"
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]
code is fine

**Review by kugamo — APPROVED** [review]
Sprites will work. APPROVED

**Pull request merged**